spec/unit/build_version_dsl_spec.rb

```diff
@@ -2,13 +2,15 @@
 
 module Omnibus
   describe BuildVersionDSL do
+    before { Config.append_timestamp(false) }
     let(:subject_with_version) { described_class.new(version_string) }
     let(:subject_with_description) { described_class.new(&description) }
 
     let(:version_string) { "1.0.0" }
     let(:description) { nil }
+    let(:today_string) { Time.now.utc.strftime('%Y%m%d') }
```

Comment:
If you fully format `today_string` with `Omnibus::BuildVersion::TIMESTAMP_FORMAT` you can use it in all the matches below: https://gist.github.com/schisamo/4c03f23c23ce0139d5ca